### PR TITLE
Upgrade user-guide to integrate maven proxy configuration (DHC-2168)

### PR DESCRIPTION
**Pull request by PierreReliquet**

### CODE REVIEW

File: modules/com.restlet/tech-doc/client/guide/04_run-tests/02_automate-tests/01_test-reports-maven.md [hunk 1]
```diff
@@ -142,6 +143,57 @@ This is done by using the following configuration:
 </code>
 </pre>
 
+### Use specific proxy configuration
+
+#### Maven proxy configuration
+
+The maven plugin can reuse the maven's installation proxy configuration simply by using the following configuration:
```
Comment (antoine-richard) [inline modules/com.restlet/tech-doc/client/guide/04_run-tests/02_automate-tests/01_test-reports-maven.md]: Maven with a capital

Comment (antoine-richard) [inline modules/com.restlet/tech-doc/client/guide/04_run-tests/02_automate-tests/01_test-reports-maven.md]: ...Maven proxy configuration**, defined in your settings.xml file,** simply...

**Comment by quilicicf** [comment]
For the environment part I think we dive too deep in explanations. It should look more like: 

> If you use environments ([see how powerful environments are](https://rest-let.com/documentation/client/user-guide/perform-requests/environment-variables)), then you need to tell the maven-plugin which one to use to run your test. Put its name (as set in the UI) in the variable `selectedEnvironment` as described below gna gna gna.

File: modules/com.restlet/tech-doc/client/guide/04_run-tests/02_automate-tests/01_test-reports-maven.md [hunk 2]
```diff
@@ -142,6 +143,57 @@ This is done by using the following configuration:
 </code>
 </pre>
 
+### Use specific proxy configuration
+
+#### Maven proxy configuration
```
Comment (PierreReliquet) [inline modules/com.restlet/tech-doc/client/guide/04_run-tests/02_automate-tests/01_test-reports-maven.md]: I agree but I am not sure to understand which order you'd like to have:

1. System
2. Maven
3. JVM

or 

1. JVM
2. Maven
3. System

**Pull request merged**